.github/workflows/scorecard.yml

```diff
@@ -10,7 +10,7 @@ on:
   # To guarantee Maintained check is occasionally updated. See
   # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
   schedule:
-    - cron: '34 11 * * 3'
+    - cron: '0 0 * * 6'
```

Comment:
could you put a comment above about this time so that it is easier to read? something like
# schedule at 0 hour at Saturday

Reply:
Good idea. I'll add that comment